.pre-commit-config.yaml

```diff
@@ -13,6 +13,11 @@ repos:
       - id: detect-private-key
       - id: end-of-file-fixer
       - id: trailing-whitespace
+  - repo: https://github.com/asottile/pyupgrade
```

Comment:
Does pyupgrade add anything over ruff?

Reply:
Looks like it doesn't anymore 😄 

https://github.com/astral-sh/ruff/issues/827